doc/api/next_api_changes/2019-03-04-AL.rst

```diff
@@ -13,6 +13,41 @@ without an explicit call to `Axes.autoscale_view`.
 In some cases, this can result in different limits being reported.  If this is
 an issue, consider triggering a draw with `fig.canvas.draw`.
 
+Autoscaling changes for Collections
+```````````````````````````````````
+
+Autoscaling has also changed for artists that are based on the `.Collections`
+class.  Previously, the method that calculates the automatic limits
+`.Collections.get_datalim` tried to take into account the size of objects
+in the collection and make the limits large enough to not clip any of the
+object.  i.e. for `.Axes.scatter` it would make the limits large enough to not
+clip any markers in the scatter.  This is problematic when the object size is
+specified in physical space, or figure-relative space, because the transform
+from physical units to data limits requires knowing the data limits, and
+becomes invalid when the new limits are applied.  This is an inverse
+problem that is theoretically solvable (if the object is physically smaller
+than the axes), but the extra complexity was not deemed worth it, particularly
+as the most common use case is for markers in scatter that are usually small
+enough to be accommodated by the default data limit margins.
+
+While the new behavior is algorithmically simpler, it is conditional on
+properties of the `.Collections` object:
+
+  1. ``offsets = None``, ``transform`` is a child of `.transData`: use the paths
```

Comment:
Doesn't link; probably clearer to mention the class, too:
```suggestion
  1. ``offsets = None``, ``transform`` is a child of `.Axes.transData`: use the paths
```
(I'm not sure if it will link with the class.)